text/0000-try-expr.md

```diff
@@ -218,18 +233,22 @@ This is our choice of keyword, because it:
 
 1. has a massive dominance in both popular and less known languages and is
    sufficiently semantically faithful to what `try` means in those languages.
+   Thus, we can leverage people's intuitions and not spend too much of our
+   complexity budget.
 2. is consistent with the standard library wrt. `Try` and `try_` prefixed methods.
-3. it is brief.
-4. it has high fidelity wrt. the concepts it attempts to communicate (exception boundary for `?`).
+3. it is brief. 
+4. it has high fidelity wrt. the concepts it attempts to communicate
+   (exception boundary for `?`). This high fidelity is from the perspective of
+   a programmers intent, i.e: "I want to try a bunch of stuff in this block".
 5. it can be further extended with `catch { .. }` handlers if we wish.
 
 ## Alternative: reserving `catch`
 
-1. **Fidelity to the construct's actual behavior:** Low
-2. **Precedent from existing languages:** None
+1. **Fidelity to the construct's actual behavior:** High
+2. **Precedent from existing languages:** Erlang and Tcl, see [prior-art]
 3. **Brevity / Length:** 6
-4. **Consistency with the naming of the trait used for `?`:** Inconsistent
-5. **Consistency with related libstd fn conventions:** Somewhat consistent
+4. **Consistency with related libstd fn conventions:** Somewhat consistent
+5. **Consistency with the naming of the trait used for `?`:** Inconsistent
```

Comment:
catch for a "try operator" seems fine.

Reply:
I'll add some nuance / clarify.

Reply:
Except that few seem to like that name ("try operator"). At the moment Rust docs calls it the *question mark operator*:
[a-shortcut-for-propagating-errors] (https://doc.rust-lang.org/book/second-edition/ch09-02-recoverable-errors-with-result.html#a-shortcut-for-propagating-errors-)

Reply:
Right; There's also some ongoing discussion about renaming the operator in documentation, etc.

Reply:
Are there other places where this is discussed besides the internals post from a day ago?

Reply:
No, I'm referring to the internals thread.